src/twisted/conch/ssh/keys.py

```diff
@@ -624,6 +710,39 @@ def _fromDSAComponents(cls, y, p, q, g, x=None):
         return cls(keyObject)
 
 
+    @classmethod
+    def _fromECComponents(cls, x, y, curve, private_value=None):
+        """
+        Build a key from EC components.
+
+        @type x: L{int}
```

Comment:
Just a minor style thing - we usually put `@param` (or `@ivar`, etc) before `@type`.  It's not a required part of the coding standard but it's probably best to do it just for consistency.